src/tree/registries/UnifiedRegistryTreeDataProvider.ts

```diff
@@ -130,4 +131,32 @@ export class UnifiedRegistryTreeDataProvider implements vscode.TreeDataProvider<
 
         void this.refresh();
     }
+
+    public async getAllConnectedRegistries(): Promise<UnifiedRegistryItem<CommonRegistry>[]> {

```

Comment:
this can be pretty slow if the user does not filter their azure subscriptions, i can definitely optimize this but should we?